language/transaction-builder/src/lib.rs

```diff
@@ -116,6 +116,15 @@ encode_txn_script! {
           `MintCapability` stored under their account and `preburn_address` has a pending burn request"
 }
 
+encode_txn_script! {
```

Comment:
Would you mind adding a basic test for this to `e2e-tests/src/tests/transaction-builder.rs`? I try to write a test for every script there to make sure the types of the Rust `encode_`  function stay in sync with the Move tx script (have  been saved by this several times while refactoring).